### ISSUE TITLE
Open API key does not store after Restart IDEA

### ISSUE THREAD

**Issue by abn-dev-01** [carlrobertoh/ProxyAI#1026]
### What happened?

3.3.0-241.1 version. 
I added Open API key - it works, 
then restarted IDEA, no API key - I have to enter it again and again after each restart.

---
IntelliJ IDEA 2025.1.1.1 (Ultimate Edition)
Build #IU-251.25410.129, built on May 9, 2025
Source revision: b815cfdcaa594
Licensed to An Nik
Subscription is active until February 9, 2026.
Runtime version: 21.0.6+9-b895.109 amd64 (JCEF 122.1.9)
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Toolkit: sun.awt.windows.WToolkit
Windows 11.0
Kotlin plugin: K2 mode
GC: G1 Young Generation, G1 Concurrent GC, G1 Old Generation
Memory: 6144M
Cores: 12
Registry:
  ide.experimental.ui=true
  jvm.dfa.analysis.ui.security.analysis.window.was.shown=true
  editor.minimap.enabled=true
Non-Bundled Plugins:
  com.github.antipovandrey.generator (1.0.2)
  com.alexanderpa.flyway.migration.creator (2.4)
  com.smallcloud.codify (6.4.1)
  com.github.glennlefevere.stenciljswebcomponents (2.1.2)
  com.rmondjone.jsontotypescript (2025.1.1)
  com.vermouthx.xcode-theme (1.7.0)
  com.guohanlin.JsonToAnyLanguage (2025.1.1)
  lermitage.intellij.extratci (2025.1.6)
  io.gitlab.zlamalp.arc-theme-idea (2.0)
  com.marcherdiego.json.visualizer (1.0.0)
  Gradle View (4.0.0)
  some.awesome (1.16)
  com.github.shlaikov.intellijbpmn2plugin (1.0.0)
  com.obroom.plugin.smartsearch (2023.1.1)
  lermitage.intellij.ilovedevtoys (1.11.0)
  light.flat.theme (0.14)
  to-go-struct (1.0.7)
  io.github.fstaudt.helm (0.10.1)
  com.flop.RestTester (2.1.0)
  wu.seal.tool.jsontokotlin (3.7.6)
  dev.priporov.idea.notes (1.5.6)
  ru.mobiledev.plugins.uigd (1.2)
  com.layoutmanager (1.5.0)
  org.jetbrains.plugins.go (251.25410.59)
  eu.oakroot.go-imports-tidy (1.1.3)
  SequenceDiagram (4.0.4-1)
  Eclipse theme (1.0)
  com.robohorse.robopojogenerator (2.6.3)
  org.intellij.RegexpTester (2.2.0)
  org.jetbrains.intellij.scripting-ruby (1.3)
  com.bartoszadamczyk.glowing-darcula (1.1.7)
  com.lilittlecat.intellij-chatgpt (1.1.1)
  net.sutanto.go_field_tag (20.04.29)
  com.jpospisil.gototabs (1.4)
  com.jsonts.kotlin (1.0.12)
  com.kondratevpavel.plugins.goland.gomethodgenerator (1.0)
  com.github.izhangzhihao.intellijgooglesearch (0.0.2)
  name.kropp.intellij.makefile (251.25410.28)
  zielu.cleansheet (211.4.2+233)
  https://github.com/DUNNIK (1.7)
  lauvsong.refactorgpt (1.0.0)
  org.ermadan.kafkalytic (0.10.5)
  StringToolsPlugin (4.22)
  org.intellij.plugins.hcl (251.25410.59)
  PythonCore (251.25410.129)
  com.crunch42.openapi (2.02)
  com.maaxgr.intellij.jsonviewer (1.0.19)
  com.bee.plugin.killGradle (1.1)
  com.github.b3er.idea.plugins.arc.browser (0.32)
  io.aesy.regex101 (0.2)
  com.obroom.plugin.toolset (2025.2.1)
  ru.adelf.idea.dotenv (251.23774.318)
  com.github.deeepamin.gitlabciaid (1.7.0)
  IDEA Restart (1.0)
  DevKit (251.25410.59)
  com.intellij.jvm.dfa.analysis (251.25410.59)
  woaini.liquibase.plugin (1.3-SNAPSHOT)
  net.hexar.json2pojo (1.1.0)
  com.cherep_corp.plugins.request_converter (1.1.4)
  Builder Generator (1.4.0)
  no.hvl.tk.visualDebugger (2.3.6)
  com.zt.json-editor (2.9.1)
  com.gustavo.swagger.generate (0.0.8)
  idv.freddie.plugin.decompiler (0.7.0)
  cn.bigcoder.plugin.objecthelper (1.3.3)
  com.github.camork.fileExpander (2.5)
  com.sunny.plugin.MockitoGenPlugin (1.5.4)
  com.xstudio.com.xstudio.plugin.idea (1.0.19)
  com.kn.diagrams.generator.generator (2022.2.0)
  com.adgainai.springbooturlfullpath (1.0.8)
  com.youssef.tool.idea.plugins (1.6)
  com.github.kisstkondoros.codemetrics (1.0.4)
  cn.renyuzhuo.plugin.easygradle (1.0.1)
  idea.plugin.tabsession (0.2)
  com.github.lppedd.idea-return-highlighter (0.8.3)
  com.mroche.JsonHelper (1.3.2)
  org.jetbrains.plugins.go-template (251.23774.318)
  Google Code Search (1.1.0)
  com.aegamesi.java_visualizer.plugin (2.2.1)
  cn.jxzhang.plugin.json-formatter (1.6)
  com.mirkoalicastro.mavenversionrefactor (1.0.5)
  com.intellij.ideolog (251.25410.24)
  VisualVMLauncher (1.23.1-IJ2023.3)
  PlantUML integration (7.11.2-IJ2023.2)
  ink.organics.pojo2json (2.1.2)
  com.jetbrains.jax.ws (251.23774.318)
  com.intellij.spring.shell (251.23774.318)
  String Manipulation (9.15.0)
  tarn.gradle.dependencies (2.0.2)
  com.redhat.devtools.intellij.telemetry (1.2.1.62)
  com.codegpt.CodeGPT (2.3.3-stable)
  jd-intellij (0.8.1)
  com.intellij.spring.webflow (251.25410.59)
  com.kimen.plugin (1.3.6)
  com.intellij.properties.bundle.editor (251.23774.318)
  CMD Support (1.0.5)
  de.docs_as_co.intellij.plugin.diagramsnet (0.2.6)
  com.intellij.spring.websocket (251.23774.318)
  Key Promoter X (2024.2.2)
  com.jetbrains.grayTheme (1.3)
  dev.pnbarx.idea.treecolor (1.2.2)
  dev.turingcomplete.intellij-gradle-utilities-plugin (1.2.5)
  com.didalgo.chatgpt (1.2.0-231)
  de.achimonline.github_markdown_emojis (1.5.0)
  io.aesy.yamllint (0.4)
  com.intellij.spring.batch (251.23774.318)
  com.intellij.aqua (251.25410.109)
  GrepConsole (13.2.0-IJ2023.3)
  com.intellij.spring.osgi (251.23774.318)
  Show As ... (1.0.3)
  fr.jukien.intellij.plugins.pojo-generator (2.6.0)
  com.intellij.bigdatatools.core (251.25410.129)
  com.intellij.bigdatatools.binary.files (251.23774.318)
  com.intellij.bigdatatools.rfs (251.25410.129)
  com.intellij.bigdatatools.metastore.core (251.23774.318)
  org.jetbrains.plugins.gitlab (251.25410.159-IU)
  com.github.novotnyr.maven-version-idea-plugin (5)
  com.intellij.spring.ws (251.23774.318)
  com.yiycf.plugins.mavenDependencyHelper (2.5)
  MavenRunHelper (4.29.0-IJ2022.2)
  ch.newinstance.plugin.mavendependencychecker (1.16.0)
  com.github.platan.gradle-dependencies-formatter (0.6.1)
  ru.alezhu.idea.plugins.named_argument_stubs (1.0.5.130)
  plantuml-parser (0.0.9)
  dev.turingcomplete.intellijbytecodeplugin (4.3.0)
  com.khch.scheduler (1.0.6)
  com.mallowigi.colorHighlighter (18.1.0)
  com.github.lppedd.kotlin-additional-highlighting (0.2.2)
  CodeAnalysis (1.1.5)
  com.utopia-rise.godotjvmideaplugin (0.13.1-4.4.1-IJ2025.1)
  izhangzhihao.rainbow.brackets.lite (1.3.0)
  dev.turingcomplete.intellijdevelopertoolsplugins (7.0.0)
  com.suusan2go.kotlin-fill-class (2.0.1)
  cn.bestwu.gdph (0.1.12)
  com.github.orange-guo.fill-kotlin-arguments (0.0.1)
  net.labymod.intellij.singlehotswap (2.10)
  com.github.nikolaikopernik.codecomplexity (1.6.2)
  GradleDependenciesHelper (1.16)
  org.jetbrains.plugins.github (251.25410.159-IU)
  org.jetbrains.junie (251.105.47)
  com.gitlab.msciachero.intellij-gitlab-plugin (12.13.0)
  ee.carlrobert.chatgpt (3.3.0-241.1)
  mobi.hsz.idea.gitignore (4.5.6)
  org.liamjwang.yamlconfig-idea.yamlconfig-idea (0.0.3)
  net.ashald.envfile (3.4.2)
  org.jetbrains.plugins.tinygo (0.5.18)
  com.github.maiqingqiang.goormhelper (1.3.0)
  com.github.linwancen.drawgraph (1.28.2025.05.15_00.20)
  org.sonarlint.idea (10.23.0.81344)
  zielu.gittoolbox (600.1.2+243)
  com.ypwang.plugin.go-linter (251.25410.109)
  cn.yiiguxing.plugin.translate (3.7.2)
  com.intellij.bigdatatools.kafka (251.23774.318)
  io.gitlab.zlamalp.arc-theme-idea-dark (1.0)
Kotlin: 251.25410.129-IJ

### Relevant log output or stack trace

```shell

```

### CodeGPT version

1.16.0

### Operating System

Windows
